test/vrt/index.js

```diff
@@ -0,0 +1,13 @@
+describe("PatternFly Elements", () => {
```

Comment:
@kylebuch8 Can we set up time to share how webrh architects this?  I think it would be good to have test files for each component.